lib/ansible/plugins/connection/local.py

```diff
@@ -29,6 +29,7 @@
 from ansible.module_utils._text import to_bytes, to_native, to_text
 from ansible.plugins.connection import ConnectionBase
 from ansible.utils.display import Display
+from ansible.utils.path import unfrackpath
```

Comment:
Why was only this import added? What happened to the remainder of the changes for the local connection?

On a related note, I noticed that there were no changes applied to this file in the 2.7 backport.

Reply:
the bug fixed for local only was relevant to 2.9+ , this import should not have made it in